CMakeLists.txt

```diff
@@ -71,5 +75,19 @@ install(DIRECTORY ${ASSETS_DIR} DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_N
 
 
 # CPack config
+set(CPACK_PACKAGE_NAME ${PROJECT_NAME})
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Chord naming application")
+set(CPACK_PACKAGE_VENDOR "Ters")
+set(CPACK_PACKAGE_VERSION_MAJOR ${PACKAGE_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PACKAGE_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PACKAGE_VERSION_PATCH})
+set(CPACK_PACKAGE_ARCHITECTURE "amd64")
```

Comment:
couldn't find this variable in the documentation

Reply:
got rid of it anyway